config/configuration/bundles/src/bundle/org/sakaiproject/config/bundle/default.sakai.properties

```diff
@@ -2660,6 +2660,11 @@
 # DEFAULT: 10
 #site.adminperms.sitesuntilpause=10
 
+### SAK-29000
+# Determines if the authorization fields are presented to the user in Site Info -> Edit Rosters -> Add Roster
+# DEFAULT: true
+# site.addroster.authorizationrequired=true
```

Comment:
Comment pattern in this document has been to set the commented out property to the opposite of the default.